### ISSUE TITLE
Show only cassandra keyspaces and tables in cqlsh

### ISSUE THREAD

**Issue by rkarthik007** [YugaByte/yugabyte-db#198]
Seeing postgres keyspaces and tables in cqlsh:
```
cqlsh> DESCRIBE KEYSPACES;

system_schema  system_auth  system  bookstore  test  foo

cqlsh> use test;
InvalidRequest: Error from server: code=2200 [Invalid query] message="SQL error (yb/yql/cql/ql/ptree/process_context.cc:180): Keyspace Not Found. Not found (yb/yql/cql/ql/util/ql_env.cc:296): Cannot use unknown keyspace
use test;
^^^
 (error -308)"
```

**Comment by m-iancu** [comment]
Reverted to a backwards compatibility issue, re-opening.

**Comment by robertpang** [comment]
Fixed in commit [5ff9faf](https://github.com/YugaByte/yugabyte-db/commit/5ff9faf0fcae287824ec1d9ba15aa8b75ae9cad5).
